pkgs/development/python-modules/octodns-powerdns/default.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, certifi
+, charset-normalizer
+, dnspython
+, fetchFromGitHub
+, fqdn
+, idna
+, natsort
+, octodns
+, python-dateutil
+, pythonOlder
+, pyyaml
+, requests
+, setuptools
+, six
+, urllib3
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "octodns-powerdns";
+  version = "0.0.5";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "octodns";
+    repo = "octodns-powerdns";
+    rev = "v${version}";
+    hash = "sha256-jt0+JnpCgvsoqMcC9mANX7uq2WPTiI2JQjwQi7LGWj0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    dnspython
+    fqdn
+    idna
+    natsort
+    octodns
+    python-dateutil
+    pyyaml
+    requests
+    six
+    urllib3
+  ];
+
```

```suggestion
  env.OCTODNS_RELEASE = 1;
```
https://github.com/octodns/octodns-powerdns/blob/6caaf910f20f77142ec9a1ce6c7a2b8ac712caa2/setup.py#L22-L25